src/data_visualization/src/main.rs

```diff
@@ -1,26 +1,76 @@
-use std::{error::Error, process};
-
-use polars::frame::DataFrame;
-use polars::prelude::*;
+use std::error::Error;
 
 mod histogram;
 mod pair_plot;
 mod parse_cols;
 mod scatter_plot;
 
+use clap::{arg, command, value_parser};
+
 fn main() -> Result<(), Box<dyn Error>> {
-	let filename = "./ressources/dataset_train.csv";
-	let Ok(data) = parse_cols::load_as_cols(filename) else {
+	let args = command!() // requires `cargo` feature
```

Comment:
you could have used a `struct` + macro

it would have been much less verbose

there are example in the other binary I made (`src/describe/src/main.rs`)